### PR TITLE
supersm 0.5.2 (new formula)

### PR DESCRIPTION
**Pull request by peeweep**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/supersm.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  depends_on "boost"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  test do
+    assert_match "#{version} \n", shell_output("#{bin}/supersm --version", 1)
```
Comment (carlocab) [inline Formula/supersm.rb:22]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

**Comment by carlocab** [comment]
Thanks for the PR -- this looks like a nice piece of software, but it currently does not meet the requirement for inclusion in Homebrew/core. The CI log reads:
```
supersm:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```
You should also have been shown this error when you ran `brew audit --new supersm` as requested in the PR template.

If you wish, you can host this formula in your own tap in the meantime. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Comment by peeweep** [comment]
I've actually run `brew audit  --new supersm` in my taps without any error messages. I don't see the prompt until I copy it to run in homebrew-core. Sorry.

**Comment by carlocab** [comment]
That's ok; it's an honest mistake. `brew audit` behaves differently depending on the tap (Homebrew/core checks are much stricter). Please run `brew audit` with the formula file in your local Homebrew/core repository when opening PRs in the future.

**Pull request closed without merging**